tools/generator-go-sdk/internal/generator/templater_methods_test.go

```diff
@@ -246,6 +255,7 @@ func (c pandaClient) List(ctx context.Context , id PandaPop) (result ListOperati
 			http.StatusOK,
 		},
 		HttpMethod: http.MethodGet,
+		Pager: ListCustomPager,
```

Comment:
This expects a struct, so we'd need to pass an instance of the struct in here?

```suggestion
		Pager: ListCustomPager{},
```